pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -66,6 +66,10 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 			queues.Push(queue)
 		}
 
+		if job.PodGroup.Status.Phase == "" {
+			continue
+		}
+
 		if job.PodGroup.Status.Phase == scheduling.PodGroupPending {
```

Comment:
Use job.IsPending() to replace 'job.PodGroup.Status.Phase == scheduling.PodGroupPending' here.

Reply:
don't do it, because the pending job  should be  pushed  to jobsMap.